Sources/Tools/MaxPlasmaMtls/Layers/plStaticEnvLayer.cpp

```diff
@@ -561,24 +561,24 @@ Matrix3 plStaticEnvLayer::IGetViewTM( int i )
     switch( i ) 
     {
         case kTopFace:
-            m.RotateX( -M_PI );   
+            m.RotateX( -hsConstants::pi<float> );
             break;
         case kBottomFace:
             break;
         case kLeftFace:
-            m.RotateX( -.5f * M_PI ); 
-            m.RotateY( -.5f * M_PI );
+            m.RotateX( -hsConstants::half_pi<float> );
+            m.RotateY( -hsConstants::half_pi<float> );
             break;
         case kRightFace:
-            m.RotateX( -.5f * M_PI ); 
-            m.RotateY( +.5f * M_PI );
+            m.RotateX( -hsConstants::half_pi<float> );
+            m.RotateY( +hsConstants::half_pi<float> );
             break;
         case kFrontFace:
-            m.RotateX( -.5f * M_PI ); 
-            m.RotateY( M_PI );
+            m.RotateX( -hsConstants::half_pi<float> );
+            m.RotateY( hsConstants::pi<float> );
```

Comment:
For parallelism's sake (the English kind, not the computing kind), I think either line 574 should lose the "+" or line 578 should gain a "+".